spec/filters/elasticsearch_spec.rb

```diff
@@ -1 +1,12 @@
+# encoding: utf-8
+
 require "logstash/devutils/rspec/spec_helper"
+require "logstash/plugin"
+
+describe "filter/elasticsearch" do
+
+  it "should register" do
```

Comment:
not going to force this but a "proper" test would be something like this, right?

```ruby
describe LogStash::Filters::Elasticsearch do
  describe "register" do
    it "should not raise an exception" do
      #....
    end
  end
end
```